packages/client/src/components/TransferToken/components/CoinTypeDropDown.js

```diff
@@ -1,17 +1,12 @@
-import { useContext } from 'react';
-import { Web3Context } from 'contexts/Web3';
-import { Dropdown } from 'library/components';
+import { Dropdown, Tooltip } from 'library/components';
 import Svg from 'library/Svg';
 
-const CoinTypeDropDown = ({ coinType, setCoinType, address }) => {
-  const web3 = useContext(Web3Context);
-  const { balances } = web3;
-  const balanceMap = balances[address];
-  const coinTypes = Object.entries(balanceMap).map(([key, value]) => ({
+const CoinTypeDropDown = ({ coinType, setCoinType, balances, tooltipText }) => {
+  const coinTypes = Object.entries(balances).map(([key]) => ({
     itemValue: key,
     displayText: key,
     attr: {
-      balance: balanceMap[key] ? Number(balanceMap[key]).toFixed(2) : 0,
+      balance: balances[key] && Number(balances[key]).toFixed(2),
```

Comment:
What's the intention for the change here? Do we want to show an empty string or 0 when the balances[key] is undefined?

Reply:
I'm not sure, I'll revert it!